p2p/src/authenticated/mod.rs

```diff
@@ -25,39 +25,18 @@
 //! of the set is smaller/larger than expected). It is up to the application to ensure sets are synchronized._
 //!
 //! Because this representation is so efficient/small, peers send bit vectors to each other periodically as a "ping" to keep
-//! the connection alive. Because it may be useful to be connected to multiple indexes of peers at a given time (i.e. to
-//! perform a DKG with a new set of peers), it is possible to configure this crate to maintain connections to multiple
-//! indexes (and pings are a random index we are trying to connect to).
-//!
-//! ```protobuf
-//! message BitVec {
-//!     uint64 index = 1;
-//!     bytes bits = 2;
-//! }
-//! ```
+//! the connection alive. It may be useful to be connected to multiple indexes of peers at a given time; for example to
+//! perform a DKG with a new set of peers while still be connected to an old set of peers. Thus, we allow maintaining connections
+//! to multiple indexes. The index is included in the message as a `u64`.
 //!
 //! Upon receiving a bit vector, a peer will select a random collection of peers (under a configured max) that it knows about that the
 //! sender does not. If the sender knows about all peers that we know about, the receiver does nothing (and relies on its bit vector
 //! to serve as a pong to keep the connection alive).
 //!
-//! ```protobuf
-//! message Peers {
-//!     repeated Peer peers = 1;
-//! }
-//! ```
-//!
 //! If a peer learns about an updated address for a peer, it will update the record it has stored (for itself and for future gossip).
 //! This record is created during instantiation and is sent immediately after a connection is established (right after the handshake).
 //! This means that a peer that learned about an outdated record for a peer will update it immediately upon being dialed.
 //!
-//! ```protobuf
```

Comment:
I think we should retain some list/diagram of what goes in each of these messages. A number of folks have reached out to tell me how useful these are to understanding what is going on.

Reply:
Whatever we do should probably be standardized with what is in `stream::public_key`:

https://github.com/commonwarexyz/monorepo/blob/d62e3270a3970b2c1c0ab5863e0e04e844873b23/stream/src/public_key/mod.rs#L28-L30